### PR TITLE
fix an odd precise comparison

### PR DESCRIPTION
**Pull request by makeProjectGreatAgain**
Tried to solve problem with comparing doubles to doubles using  `== `
Changed:
`left.Width.Value == right.Width.Value))`
to
`(Math.Abs(left.Width.Value - right.Width.Value) < tolerance)))`
with tolerance = 1e-10;

### CODE REVIEW

File: ClosedXML/Excel/XLWorkbook_Save.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ namespace ClosedXML.Excel
     public partial class XLWorkbook
     {
         private const Double ColumnWidthOffset = 0.710625;
-
+        private static readonly double tolerance = 1e-10;
```
Comment (Pankraty) [inline ClosedXML/Excel/XLWorkbook_Save.cs:59]: Use XLHelper.Epsilon instead

Comment (makeProjectGreatAgain) [reply ClosedXML/Excel/XLWorkbook_Save.cs]: >Have you experienced any issues caused by that comparison?

No, I haven't

>Use XLHelper.Epsilon instead

Done

**Comment by igitur** [comment]
> Have you experienced any issues caused by that comparison?

Would also like to know.

**Comment by makeProjectGreatAgain** [comment]
> > Have you experienced any issues caused by that comparison?
> 
> Would also like to know.

I already answered this above. No, I haven't. Perhaps this changes aren't necessary

**Pull request merged**